### DISCUSSION TITLE
Windows users, Docker, and a lot of frustration

### DISCUSSION THREAD

**Discussion by scaccomatto75** [FlareSolverr/FlareSolverr#1402] [Q&A]
Like many other users, I have been having problems with Flaresolverr for several weeks.
I am a windows user, and unfortunately I had never used docker before.
I tried to understand how to implement all the possible solutions that you shared in the various posts, but I think that my ignorance on the subject is too much compared to what they require.
Therefore, I ask you if it would be possible to have a small tutorial for dummies, or a setup of a beta version with the fixes that are trying to solve these problems.
Thank you all for understanding and I apologize if my request was off topic.
Mauriziov

**Comment by ilike2burnthing** [comment]
Run from source - https://github.com/FlareSolverr/FlareSolverr#from-source-code

Or, build from source - https://github.com/FlareSolverr/FlareSolverr/pull/1163#issuecomment-2067972723

Using - https://github.com/21hsmw/FlareSolverr/tree/nodriver-support

**Reply by ggocek** [reply]
I am trying to use FlareSolverrSharp with the pre-compiled binaries in Windows 11, C#, .NET 4.8.
I am trying to run the FlareSolverrSharp sample program. 
The FlareSolverr pre-compiled server starts up AOK. I can set environment variables.

"new ClearanceHandler" works.
But when the code reaches
await client.GetStringAsync(protectedUrl)
It crashes with a 404 HttpRequestException exception (the sample code does not use any try-catch blocks).
I have tried various adjustments for ProxyUrl, GetAsync instead of GetStringAsync, etc. But either the sample Get method or PostUrlEncoded method fails due to the 404.

DEBUG logging appears when I start the server, but no log lines appear in the server window while the calls are made,
My first goal is to get client.GetStringAsync(protectedUrl) to work, just for the FlareSolverrSharp sample.

**What is the 404 telling me?** I am using the FlareSolverrSharp sample URL (badasstorrents).

Am I really out of luck unless I install python (and maybe docker)? Thanks.

**Reply by ilike2burnthing** [reply]
Run https://github.com/21hsmw/FlareSolverr/tree/nodriver-support from source, following the instructions in the readme, or `21hsmw/flaresolverr:nodriver` in Docker.

**Reply by ggocek** [reply]
Thanks for your help. For my purposes, I am not going to install python and/or docker.
